prow/jobs/incubator/hack-showcase/hack-showcase.yaml

```diff
@@ -1,6 +1,6 @@
-job_template: &job_template
+base_job_template: &base_job_template
   skip_report: false
-  always_run: true
+  run_if_changed: "^hack-showcase/github-connector"
```

Comment:
hack-showcase is your repository, so it's your root in file path.
Try this one : `run_if_changed: "^github-connector"`